cultivos-tropicales.csl

```diff
@@ -0,0 +1,503 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="es-ES">
+  <info>
+    <title>Cultivos Tropicales (Spanish)</title>
+    <title-short>CulTrop</title-short>
+    <id>http://www.zotero.org/styles/cultivos-tropicales</id>
+    <link href="http://www.zotero.org/styles/cultivos-tropicales" rel="self"/>
+    <link href="http://www.zotero.org/styles/iso690-author-date-es" rel="template"/>
+    <link href="http://ediciones.inca.edu.cu/index.php/ediciones/pages/view/autores" rel="documentation"/>
+    <author>
+      <name>Rafael Cervantes Beyra</name>
+      <email>cervantes@unah.edu.cu</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="engineering"/>
+    <issn>0258-5936</issn>
+    <eissn>1819-4087</eissn>
+    <summary>Style based on ISO 690:2010(F), V1</summary>
+    <updated>2014-05-30T00:36:38+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale>
+    <terms>
+      <term name="anonymous">Anon.</term>
+      <term name="no date">[s.f]</term>
+      <term name="in">en</term>
+      <term name="online">en&#160;linea</term>
+      <term name="retrieved">disponible&#160;</term>
+      <term name="from">en</term>
+      <term name="accessed">consultado:&#160;</term>
```

Comment:
No sé qué pudo haber pasado, el fichero CSL que tengo no contiene ese carácter (que es un punto), simplemente copié y pegue. De igual modo ya lo eliminé, gracias por informarme, estaré más al tanto.

 

De: Sebastian Karcher [mailto:notifications@github.com] 
Enviado el: Tuesday, June 16, 2015 10:35 AM
Para: citation-style-language/styles
CC: Rafael Cervantes Beyra
Asunto: Re: [styles] Cultivos Tropicales (Spanish) (#1584)

 

In cultivos-tropicales.csl <https://github.com/citation-style-language/styles/pull/1584#discussion_r32547813> :

> +    <category field="engineering"/>
> +    <issn>0258-5936</issn>
> +    <eissn>1819-4087</eissn>
> +    <summary>Style based on ISO 690:2010(F), V1</summary>
> +    <updated>2014-05-30T00:36:38+00:00</updated>
> +    <rights license=" <http://creativecommons.org/licenses/by-sa/3.0/> http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
> +  </info>
> +  <locale>
> +    <terms>
> +      <term name="anonymous">Anon.</term>
> +      <term name="no date">[s.f]</term>
> +      <term name="in">en</term>
> +      <term name="online">en&#160;linea</term>
> +      <term name="retrieved">disponible&#160;</term>
> +      <term name="from">en</term>
> +      <term name="accessed">consultado:&#160;</term>

<term name="accessed">consultado:&#160;�</term>
Hay un caracter invisible (y ilegal) despues del non-breaking space. 

—
Reply to this email directly or view it on GitHub <https://github.com/citation-style-language/styles/pull/1584/files#r32547813> .  <https://github.com/notifications/beacon/AG7e4pKlvhUFujFuSGFbLyTdZqJDDG0eks5oUFWxgaJpZM4E-o5l.gif>